### PR TITLE
feat: language selector

### PR DESCRIPTION
**Pull request by Rigonkmalk**

### CODE REVIEW

**Comment by sebastiendeu** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There're changes, but no changelog.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by sebastiendeu** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There're changes, but no changelog.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: changelog/unreleased/add-language-selector.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+type: changed
```
Comment (sebastiendeu) [inline changelog/unreleased/add-language-selector.yml:2]:
```suggestion
type: added
```

File: changelog/unreleased/add-language-selector.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+title: Add language selector
```
Comment (sebastiendeu) [inline changelog/unreleased/add-language-selector.yml:3]:
```suggestion
title: Add language selector dropdown in navbar
```

**Review by vfricou — CHANGES_REQUESTED** [review]
French traductions issues :
- On section / vlan / usage edit buttun update section : « Modifier ce(tte) … »
- On usage en FR : Oublier -> Change to Relacher ou Libérer
- On section show Éteint : Change to inactif
- On section show : All top buttons aren't translated

All page /register/edit isn't translated 
Dark theme switch isn't functionnal

**Review by vfricou — DISMISSED** [review]

**Review by vfricou — APPROVED** [review]

**Pull request merged**